react/PasswordExample/styles.styl

```diff
@@ -1,5 +1,7 @@
+palette=json('../../stylus/settings/palette.json', { hash: true })
+
 .Token--number
-    color var(--dodgerBlue)
+    color palette['dodgerBlue']
```

Comment:
[nit] No `:` here? In another commit, the message says that it was mandatory to put `:` after the property name when an hash was used.

Reply:
Ah yes it does not seem necessary here, it seems like a stylus peculiarity :?